src/Service.GraphQLBuilder/GraphQLNaming.cs

```diff
@@ -203,18 +129,7 @@ public static string FormatNameForField(string name)
         /// <returns></returns>
         public static NameNode Pluralize(string name, Entity configEntity)
         {
-            if (TryGetConfiguredGraphQLName(configEntity, out string? graphQLName) &&
-                !string.IsNullOrEmpty(graphQLName))
-            {
-                return new NameNode(graphQLName.Pluralize());
-            }
-            else if (TryGetSingularPluralConfiguration(configEntity, out SingularPlural? namingRules) &&
-                !string.IsNullOrEmpty(namingRules.Plural))
-            {
-                return new NameNode(namingRules.Plural);
-            }
-
-            return new NameNode(name.Pluralize(inputIsKnownToBeSingular: false));
+            return new NameNode(configEntity.GraphQL.Plural);
```

Comment:
While it might not be stored in the config, we generate it as part of the deserialization process, in the `RuntimeEntities` constructor.

This way the `RuntimeConfig` object knows everything about how to model entities, whereas we use to have a separate object that we'd look at _or_ we'd generate it on the fly.

The result is that we only run the pluralisation logic once per entity, not (potentially) every time we need to find its plural name.